src/Bridge/NelmioApiDoc/Parser/ApiPlatformParser.php

```diff
@@ -82,11 +82,11 @@ public function parse(array $item) : array
         list($io, $resourceClass, $operationName) = explode(':', $item['class'], 3);
         $resourceMetadata = $this->resourceMetadataFactory->create($resourceClass);
 
-        $classOperations = $this->getGroupsForItemAndCollectionOperation($resourceMetadata, $operationName);
+        $classOperations = $this->getGroupsForItemAndCollectionOperation($resourceMetadata, $operationName, $io);
+        echo '<pre>';var_dump($classOperations['serializer_groups'], !empty($classOperations['serializer_groups'])); echo '</pre>';
```

Comment:
I you recall the exemple i've put in the issue, than you should understand why it is strange, because it's return positive values : 
```php
array(2) {
  [0]=>
  string(8) "dummy_in"
  [1]=>
  string(12) "custom_dummy"
}
bool(true)
```